source-http-ingest/README.md

```diff
@@ -0,0 +1,74 @@
+In order to prevent problems due to duplicate processing of webhooks, these services typically provde either an HTTP header or a field within each document that serves
+as a unique ID for each webhook event. This can be used to deduplicate the events in your `webhook-data` collection. The key of the discovered `webhook-data` collection is `/_meta/webhookId`.
+By default, this value is generated automatically by the connector, and no-deduplication will be performed.
+You can set the `idFromHeader` option in the resource configuration to have the connector automatically assign the value of the given HTTP header to the `/_meta/webhookId` property.
+Doing so means that a materialization of the `webhook-data` collection will automatically deduplicate the webhook events.
+
+Here's a table with some common webhook services and headers that they use:
+
+| Service | Value to use for `idFromHeader`  |
+|---------|----------------------------------|
+| Github  | `X-Github-Event`                 |
+| Shopify | `X-Shopify-Webhook-Id`           |
+| Zendesk | `x-zendesk-webhook-id`           |
+| Jira    | `X-Atlassian-Webhook-Identifier` |
+
+### Custom collection IDs
+
+Some webhooks don't pass a deduplication ID as part of the HTTP headers. That's fine, and you can still easily deduplicate the events.
+To do so, you'll just need to customize the `schema` and `key` of your webhook-data collection, or bind the webhook to an existing collection that already has the correct `schema` and `key`.
+Just set the `key` to the field(s) within the webhook payload that uniquely identify the event.
+For example, to capture webhooks from Segment, you'll want to set the `key` to `["/messageId"]`, and ensure that the `schema` requires that property to exist and be a `string`.
+
+### Webhook signature verification
+
+This connector does not yet support webhook signature verification. If this is a requirement for your use case, please contact [`support@estuary.dev`](mailto://support@estuary.dev) and let us know.
+
+## Endpoint Configuration reference
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| **** | EndpointConfig |  | object | Required |
+| `/require_auth_token` |  | Optional bearer token to authenticate webhook requests.<br><br>WARNING: If this is empty or unset, then anyone who knows the URL of the connector will be able to write data to your collections. | null, string | `null` |
+
+## Resource configuration reference
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| **** | ResourceConfig |  | object | Required |
+| `/idFromHeader` |  | Set the &#x2F;&#x5F;meta&#x2F;webhookId from the given HTTP header in each request.<br><br>If not set, then a random id will be generated automatically. If set, then each request will be required to have the header, and the header value will be used as the value of &#x60;&#x2F;&#x5F;meta&#x2F;webhookId&#x60;. | null, string |  |
```

Comment:
These tables look a little messed up?

Reply:
They still render correctly, though I agree it's ugly in text. These are produced by `flow-schemalate`, though, so I'd like to just address it there.